File: Solutions/1893. Check if All the Integers in a Range Are Covered/1893. Check if All the Integers in a Range Are Covered.cpp
class Solution {
public:
    bool isCovered(vector<vector<int>>& ranges, int left, int right) {
        int cnt[51]={};
        for(const auto& r:ranges)
            for(int x=r[0];x<=r[1];++x)
                cnt[x]=1;
        for(int i=left;i<=right;++i)
            if(cnt[i]==0)
                return false;
        return true;
    }
};

File: Solutions/1893. Check if All the Integers in a Range Are Covered/1893. Check if All the Integers in a Range Are Covered.py
class Solution:
    def isCovered(self, ranges: List[List[int]], left: int, right: int) -> bool:
        cnt=[0]*51
        for r in ranges:
            for x in range(r[0],r[1]+1):
                cnt[x]=1
        for i in range(left,right+1):
            if cnt[i]==0:
                return False
        return True